### DISCUSSION TITLE
Home Assistant Docker Version + RaspberryMatic auf Raspberry3 + beides verbinden

### DISCUSSION THREAD

**Discussion by martin8310** [OpenCCU/OpenCCU#1806] [🇩🇪 Fragen&Antworten] [answered]
Hallo,

wie bekomme ich das RaspberryMatic Add-On auf eine Home Assistant  Docker Version?
Oder anders gesagt, wie verbinde ich Home Assistant Docker (läuft auf einem PC) mit dem Raspberry PI 3 auf dem RaspberryMatic läuft.

Gruß und Dank 
Martin

**Comment by jens-maus** [comment, marked as answer]
> wie verbinde ich Home Assistant Docker (läuft auf einem PC) mit dem Raspberry PI 3 auf dem RaspberryMatic läuft.

Siehe Dokumentation:
https://github.com/jens-maus/RaspberryMatic/wiki/HomeAssistant-Integration

**Reply by martin8310** [reply]
Hallo, 

danke für die schnelle Antwort. Ich denke Du meinst die Anleitung "Home Assistant Community Store (HACS) Setup" auf deiner genannten Seite. Leider geht das nicht, da die Homeassistant-Docker Version keinen Addon-Store hat. Die Addons gibt es nur in den Versionen OS und Supervised (siehe https://www.home-assistant.io/installation).

Vrstehe ich das richtig, oder übersehe ich etwas?

Gruß
Martin

**Reply by jens-maus** [reply]
> Vrstehe ich das richtig, oder übersehe ich etwas?

Kann gut sein. Überblicke ich gerade selbst auch nicht. Würde ich ohnehin so nicht betreiben und immer auf HomeAssistantOS oder Supervised setzen. Alles anderes macht IMHO heutzutage keinerlei Sinn mehr.

**Reply by martin8310** [reply]
Hallo, danke für die Antwort. Keine Ahnung was IMHO bedeutet, aber ich denke Docker ist die Zukunft. Denn man kann endlich mal eine Anwendung unter Linux nutzen, die kein stundenlanges Pakete nachinstallieren verursacht. Mit Docker ist es halt so einfach eine Anwendung zu nutzen wie unter Windows, also perfekt ;-) Wer ein Synology NAS nutzt, ist mit Dockern auch gut bedient.

Ich werde dann mal abwarten, vielleicht finde ich noch eine passende Lösung für mich, danke ;-)

Gruß
Martin

**Reply by gerritwalther** [reply]
Hallo Martin, konntest du eine Lösung finden? Habe den gleichen Ansatz mit Containern gewählt, da ich noch Octoprint auf dem Raspberry Pi laufen habe. Ich habe RaspberryMatic in meine docker-compose.yml hinzugefügt und es startet auch, bei dem Schritt komme ich aber gerade nicht weiter und mir fehlt der Schritt zur Integration in HA.

**Reply by SukramJ** [reply]
HACS ist kein AddOn sondern eine custom_component, entsprechend braucht man auch keinen AddOn-Store.

Die HA supervised Installation ist eine Container basierte HA Installationsmethode wobei ich die Installation über HAOS auch Container basiert empfehlen würde.

Mit ein wenig Transferleistung kriegt man mit der [erstklassigen Anleitung](https://github.com/jens-maus/RaspberryMatic/wiki/HomeAssistant-Integration) von Jens Maus auch eine Installation von HACS unter Docker hin.
Ansonsten sind über Google natürlich auch weitere HACS [Installationsanleitungen](https://letmegooglethat.com/?q=hacs+installieren+docker) zu finden.

**Comment by distancerunner** [comment]
Ich habe HomeAssist nun auch als Docker Container am laufen. Dann bin ich in das Volume gegangen und habe dort einen Ordner custom_components erstellt. Dann mein Addon in den Ordner kopiert und den Container neu gestartet. Schwupps, war das Addon nutzbar hinzugefügt.
